examples/high_frequency/emc/busbar.py

```diff
@@ -117,18 +117,66 @@
 print(q3d.net_sources("Bar2"))
```

Comment:
@tizianrot here and above these lines there are a series of prints, do you think are really needed?